sky/serve/service_spec.py

```diff
@@ -233,6 +237,8 @@ def add_if_not_none(section, key, value, no_empty: bool = False):
                         self.upscale_delay_seconds)
         add_if_not_none('replica_policy', 'downscale_delay_seconds',
                         self.downscale_delay_seconds)
+        add_if_not_none('service', 'load_balancing_policy',
+                        self._load_balancing_policy)
```

Comment:
```suggestion
        add_if_not_none('load_balancing_policy', None
                        self._load_balancing_policy)
```
should this be none? you can try to debug with from_yaml_config(service.to_yaml_config()) and checked if everying is the same.

Reply:
True, good catch! 👍